### DISCUSSION TITLE
Package maintainence/updates

### DISCUSSION THREAD

**Discussion by snaerth** [MichalLytek/type-graphql#1182] [Q&A]
First of all thank you all for you amazing work with this package/library.

Is this repository slowly dying? I see there have been some updates but not a release since 3.Nov 2020.
I am thinking of switching to Nest.js for better maintainability since my backend heavily relies on this package.

It would be great if I can get some clarification on future maintenance

**Comment by MichalLytek** [comment]
> but not a release since 3.Nov 2020.

https://www.npmjs.com/package/type-graphql/v/1.2.0-rc.1
> 1.2.0-rc.1
> Published 4 months ago

> Is this repository slowly dying?

https://github.com/MichalLytek/type-graphql/branches
> v16
> Updated 7 days ago by MichalLytek

Please take a look at issues and discussion before asking such questions.
I don't have enough time to work on some big features, but I respond to all questions really fast.

**Reply by snaerth** [reply]
@MichalLytek thx for the answer. I will take a better look next time.
